src/themes/eLife/styles.css

```diff
@@ -300,3 +310,93 @@ blockquote {
     line-height: 1.56522;
   }
 }
+
+:--cite-group {
+  list-style: none;
+  counter-reset: cite-counter;
+  margin: 0;
+  padding: 0;
+  font-family: var(--heading-font);
+
+  ol,
+  ul {
+    padding-left: 0;
+    list-style: none;
+  }
+
+  > li {
+    counter-increment: cite-counter;
+    font-size: 0.875rem;
+    line-height: 1.71429;
+    position: relative;
+    margin-bottom: 0.75rem;
+    padding-left: 3rem;
+
+    &::before {
+      content: counter(cite-counter);
+      display: inline-block;
+      text-align: right;
+      font-size: 1rem;
+      font-weight: 600;
+      position: absolute;
+      margin: 0 -2rem;
+    }
+  }
+
+  :--creative-work > :--title {
+    display: inline-block;
+    font-size: 1rem;
+    line-height: 1.5;
+    font-weight: 600;
+    border-bottom: none;
+  }
+
+  :--authors {
+    display: inline;
+
+    :--person {
+      display: inline;
+
+      a {
+        border-bottom: none;
+
+        &::after {
```

Comment:
Do `:--authors :--person` always have an `anchor` element as children, or can a Person have a non-linked text for the name?